slack-first/overlays/moc/smaug/config.yaml

```diff
@@ -1,17 +1,25 @@
 supportChannelId: C01RMPVUUK1
 onCall:
-  schedule: weekly
+  schedule: daily
   members:
-    - slack: U01SKRUKCM6
-      github: 4n4nd
     - slack: U02580V036C
       github: Gregory-Pereira
     - slack: U01SW975LQY
       github: harshad16
     - slack: U01RF5SASLX
       github: HumairAK
-    - slack: U02C0MN9YDC
-      github: dystewart
+    - slack: U01S8JVDS65
+      github: durandom
+    - slack: U02CWCX960N
+      github: schwesig
+    - slack: U01S22XJD60
+      github: tumido
+    - slack: U0277G1VAAV
+      github: mimotej
+    - slack: U02HA8W357G
+      github: mayaCostantini
```

Comment:
I do not see `mayaCostantini` on this list. And @VannTen (Max)  seems to be missing.

Reply:
ACK

Reply:
For now I have removed maya and added max if there will be any other changes needed I will add them :+1:

Reply:
@harshad16 we decided with @goern you should not be on call, but act as a thoth devops